be/src/storage/tablet_updates.cpp

```diff
@@ -77,7 +77,10 @@ std::string EditVersion::to_string() const {
     }
 }
 
-TabletUpdates::TabletUpdates(Tablet& tablet) : _tablet(tablet), _unused_rowsets(UINT64_MAX) {}
+TabletUpdates::TabletUpdates(Tablet& tablet) : _tablet(tablet), _unused_rowsets(UINT64_MAX) {
+    _level_multiple = config::size_tiered_level_multiple;
+    _max_level_size = config::size_tiered_min_level_size * pow(_level_multiple, config::size_tiered_level_num);
```

Comment:
Why make `_level_multiple` and `_max_level_size` as fields in class `TabletUpdates` ?

Reply:
They will waste memory space, these configurations cannot be dynamically modified